scripts/src/3rdparty.lua

```diff
@@ -1956,88 +1959,94 @@ project "asmjit"
 		end
 	end
 
+	if (_OPTIONS["PLATFORM"]=="arm" or _OPTIONS["PLATFORM"]=="arm64") then
+		configuration { "x64" }
```

Comment:
Careful: this also applies the configuration filter to the file list below. You could reset it after the defines block, but I don't believe this line is actually needed.